source/markuspline_XL.patch

```diff
@@ -0,0 +1,34 @@
+--- module_markuspline.f90	2017-06-30 18:45:06.638394848 +0200
```

Comment:
This file patches your markuspline to be IBM XL compilable. Doesn't change the actual working of the code. I would recommend to apply it to markuspline repo instead and remove the patch.